src/domain/app/appWideNotification/appWideNotificationConstants.ts

```diff
@@ -0,0 +1,29 @@
+import { normalizeActionName } from "../../utils";
+
+export const AppWideNotificationActions = {
+  FETCH: normalizeActionName("appWideNotification/FETCH"),
+  RECEIVE: normalizeActionName("appWideNotification/RECEIVE"),
+  FETCH_ERROR: normalizeActionName("appWideNotification/FETCH_ERROR"),
+};
+
+export type AppWideNotificationState = {
+  isFetching: boolean;
+  data: Array<object>;
```

Comment:
Typing in this could be Array<AppWideNotificationObject>

Reply:
Fixed typing